easybuild/easyconfigs/l/lsb_release/lsb_release-2.0.eb

```
@@ -0,0 +1,28 @@
+easyblock = 'ConfigureMake'
+
+name = 'lsb_release'
+version = '2.0'
+local_cimmit = 'fd23738'
```

Is this a typo? Or was this for a reason?
```suggestion
local_commit = 'fd23738'
```